etcdserver/snapshot_store.go

```diff
@@ -28,6 +30,12 @@ import (
 	dstorage "github.com/coreos/etcd/storage"
 )
 
+// Because raft state machine retries to send snapshot to slow follower when
+// receiving MsgHeartbeatResp from the follower, the newly created
+// snapshot should be gotten within one heartbeat interval.
+// Set it as 5s to match the upper limit of heartbeat interval.
+const closeUnusedSnapshotInterval = 5 * time.Second
```

Comment:
This comment misses the first part, which i believe is the most important one in many cases.